README.md

```diff
@@ -1,38 +1,63 @@
-```gfm
-# insert project logo - replace the below logo location with the location of your logo
-![](https://github.com/lfai/artwork/blob/master/projects/PROJECT NAME/PROJECT NAME-color.svg)
-```
-
-```gfm
-# Add badges that point to your LICENSE, CII status, and build environment (if it exists). Check out other badges to add at https://shields.io/
-![GitHub](https://img.shields.io/github/license/lfenergy/PROJECT NAME)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/CII ID/badge)](https://bestpractices.coreinfrastructure.org/projects/CII ID)
-```
-
-# PROJECT NAME
-
-_2-3 sentence description of the project_
-
-## Install
-
-_Provide install instructions here or point to an INSTALL.md file_
-
-## Build
-
-_Provide build instructions here or point to an BUILD.md file_
-
-## Discussion
-
-You can connect with the community in a variety of ways...
-
-- [LINK TO MAILING LIST](https://lists.lfaidata.foundation/g/xxxx-discussion)
-- [#PROJECT NAME channel on LF AI and Data Slack](https://slack.lfaidata.foundation)
-
-## Contributing
-Anyone can contribute to the PROJECT NAME project - learn more at [CONTRIBUTING.md](CONTRIBUTING.md)
-
-## Governance
-PROJECT NAME is a project hosted by the [LF AI and Data Foundation](https://lfaidata.foundation). This project's techincal charter is located in [CHARTER.md](tsc/CHARTER.md) and has established it's own processes for managing day-to-day processes in the project at [GOVERNANCE.md](GOVERNANCE.md).
-
-## Reporting Issues
-To report a problem, you can open an [issue](https://github.com/lfai/PROJECT NAME/issues) in repository against a specific workflow. If the issue is senstive in nature or a security related issue, please do not report in the issue tracker but instead email PROJECT NAME-private@lists.lfaidata.foundation.
+# OpenDataology
+
+----
+
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6032/badge)](https://bestpractices.coreinfrastructure.org/projects/6032) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+## Overview
+
+----
+
+OpenDatalogy is an open source dataset license compliance analysis project. Our project enables users of publicly available datasets and users who curate a dataset from multiple data sources (particularly for use as a part of machine learning models) to identify the potential license compliance risks. Our project is primarily comprised of three key components. 

```

Comment:
OpenDataology is a project for AI model trainning with trusted dataset compliance.